### DISCUSSION TITLE
VDataTable doesn't allow the ability to override the `tbody` element

### DISCUSSION THREAD

**Discussion by craigrileyuk** [vuetifyjs/vuetify#17451] [Ideas]
https://github.com/vuetifyjs/vuetify/blob/0c4729276eadd7f57c3cfb6340d0bb709e8d6b4a/packages/vuetify/src/labs/VDataTable/VDataTable.tsx#L216-L224

Previously, the way to implement something like table row animation was to replace the `tbody` HTML element with a `transition-group`. However, the `body` slot is nested inside `tbody` and the `tbody` slot renders after the original tbody.

Given that `VDataTableHeaders` isn't exported, there's no way to re-implement the table functionality easily with the default slot either.

Ideally, the default content for the `body` slot should encompass the `tbody` tag as well.

**Comment by kn0x1c** [comment]
I have the same issue, currently I am not able to animate the table content, which is not ideal...
